_data/retail.yml

```diff
@@ -31,6 +31,11 @@ websites:
       tfa: No
       twitter: comixology
 
+    - name: Computer Alliance
+      url: http://www.computeralliance.com.au/
+      img: computeralliance.png
+      tfa: No
+
```

Comment:
I suggest `facebook: ComputerAllianceAus` as their Twitter doesn't seem to be monitored by them.